Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM ocaml/opam:alpine-3.21-ocaml-5.2 AS build
+FROM alpine:3.21 AS build
 
 # Install system dependencies
-RUN sudo apk -U upgrade --no-cache && sudo apk add --no-cache \
+RUN apk -U upgrade --no-cache && apk add --no-cache \
+    # to download and install Dune Developer Preview with alpine:3.21
+    build-base patch tar ca-certificates git rsync curl sudo bash \
```

Comment:
Is there a need for `sudo`, `rsync`, `bash` or `nano`?

Reply:
Indeed, but dune uses curl internally for `(url ..)` fields, so we still need it.